### PR TITLE
Edit docs

### PR DESCRIPTION
**Pull request by andy9t7**
I have transferred the contents from slicercbm.org documentation to this one with some structuring adjustments and a new webpage theme.

### CODE REVIEW

**Comment by andy9t7** [comment]
Error building on readthedocs server:
```
Running Sphinx v6.2.1

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/slicercbm/envs/edit-docs/lib/python3.9/site-packages/sphinx/registry.py", line 442, in load_extension
    mod = import_module(extname)
  File "/home/docs/.asdf/installs/python/3.9.15/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'myst_parser'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/slicercbm/envs/edit-docs/lib/python3.9/site-packages/sphinx/cmd/build.py", line 280, in build_main
    app = Sphinx(args.sourcedir, args.confdir, args.outputdir,
  File "/home/docs/checkouts/readthedocs.org/user_builds/slicercbm/envs/edit-docs/lib/python3.9/site-packages/sphinx/application.py", line 229, in __init__
    self.setup_extension(extension)
  File "/home/docs/checkouts/readthedocs.org/user_builds/slicercbm/envs/edit-docs/lib/python3.9/site-packages/sphinx/application.py", line 404, in setup_extension
    self.registry.load_extension(self, extname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/slicercbm/envs/edit-docs/lib/python3.9/site-packages/sphinx/registry.py", line 445, in load_extension
    raise ExtensionError(__('Could not import extension %s') % extname,
sphinx.errors.ExtensionError: Could not import extension myst_parser (exception: No module named 'myst_parser')

Extension error:
Could not import extension myst_parser (exception: No module named 'myst_parser')
```

**Comment by andy9t7** [comment]
> Error building on readthedocs server:
> 
> ```
> Running Sphinx v6.2.1
> 
> Traceback (most recent call last):
>   File "/home/docs/checkouts/readthedocs.org/user_builds/slicercbm/envs/edit-docs/lib/python3.9/site-packages/sphinx/registry.py", line 442, in load_extension
>     mod = import_module(extname)
>   File "/home/docs/.asdf/installs/python/3.9.15/lib/python3.9/importlib/__init__.py", line 127, in import_module
>     return _bootstrap._gcd_import(name[level:], package, level)
>   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
>   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
>   File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
> ModuleNotFoundError: No module named 'myst_parser'
> 
> The above exception was the direct cause of the following exception:
> 
> Traceback (most recent call last):
>   File "/home/docs/checkouts/readthedocs.org/user_builds/slicercbm/envs/edit-docs/lib/python3.9/site-packages/sphinx/cmd/build.py", line 280, in build_main
>     app = Sphinx(args.sourcedir, args.confdir, args.outputdir,
>   File "/home/docs/checkouts/readthedocs.org/user_builds/slicercbm/envs/edit-docs/lib/python3.9/site-packages/sphinx/application.py", line 229, in __init__
>     self.setup_extension(extension)
>   File "/home/docs/checkouts/readthedocs.org/user_builds/slicercbm/envs/edit-docs/lib/python3.9/site-packages/sphinx/application.py", line 404, in setup_extension
>     self.registry.load_extension(self, extname)
>   File "/home/docs/checkouts/readthedocs.org/user_builds/slicercbm/envs/edit-docs/lib/python3.9/site-packages/sphinx/registry.py", line 445, in load_extension
>     raise ExtensionError(__('Could not import extension %s') % extname,
> sphinx.errors.ExtensionError: Could not import extension myst_parser (exception: No module named 'myst_parser')
> 
> Extension error:
> Could not import extension myst_parser (exception: No module named 'myst_parser')
> ```

Resolved the error.

URL to check: https://slicercbm.readthedocs.io/en/edit-docs/

File: .DS_Store
Comment (benzwick) [inline .DS_Store:1]: I don't think these files should be added. Maybe add this to .gitignore.

Comment (andy9t7) [reply .DS_Store:1]: I have removed all the DS_Store files and added to .gitignore.

**Pull request merged**